### PR TITLE
Brief explanation about ' (quote)

### PR DESCRIPTION
**Pull request by coelhotopetudo**
Used https://github.com/adambard/learnxinyminutes-docs/blob/f835be4f02a6a855d4ade9f296fa29aaa71b647e/common-lisp.html.markdown as reference.

### CODE REVIEW

File: elisp.html.markdown [hunk 1]
```diff
@@ -225,6 +225,8 @@ filename: learn-emacs-lisp.el
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;
 ;; Let's store a list of names:
+;; If you want to create a literal list of data, use ' to stop it from
```
Comment (vendethiel) [inline elisp.html.markdown]: If -> When?

Comment (coelhotopetudo) [inline elisp.html.markdown]: yeah

**Comment by vendethiel** [comment]
Thanks!

**Comment by vendethiel** [comment]
I know Common Lisp has this, so maybe Elisp as well... Is it considered OK to do mutating operations on such quoted constructs?

**Pull request merged**